Makefile

```diff
@@ -62,6 +63,33 @@ lint:
 release-major release-minor release-patch:
 	@$(XYZ) --increment $(@:release-%=%)
 
+.PHONY: bundle
+bundle:
+	mkdir dist || echo "Directory already exists"
+	cat \
+  node_modules/sanctuary-type-identifiers/index.js \
+  node_modules/sanctuary-type-classes/index.js \
+  node_modules/sanctuary-def/index.js \
+  index.js \
+  >dist/sanctuary.js
+	cat dist/sanctuary.js | $(UGLIFYJS) -mc > dist/sanctuary.min.js
```

Comment:
This paragraph can now be deleted since (below) we state that `bundle` depends on `dist/sanctuary.js` and `dist/sanctuary.min.js`.

Reply:
Is this particular change resolved?